src/pornin.rs

```diff
@@ -0,0 +1,415 @@
+use core::cmp::PartialEq;
```

Comment:
Re: the file name, I forgot about this in the previous PR but I'll probably change this to "legendre_pornin" and "inversion_bernsteinyang" so the functionality of the file is evident.

Reply:
Ok, but "jacobi_pornin" is preferable.